uint16 id
bool static
geometry_msgs/Pose pose
geometry_msgs/Pose future_pose1
geometry_msgs/Pose future_pose2
float32 width
bool exited
